test/e2e/data/infrastructure-docker/v1beta1/clusterclass-quick-start.yaml

```
@@ -39,20 +39,27 @@ spec:
       openAPIV3Schema:
         type: string
         default: "k8s.gcr.io"
+        example: "k8s.gcr.io"
+        description: "imageRepository is the container registry to pull images from. If empty, `k8s.gcr.io` will be used by default."
   - name: etcdImageTag
     required: true
     schema:
       openAPIV3Schema:
         type: string
         default: ""
+        example: "3.5.1"
+        description: "etcdImageTag is the tag for the etcd image."
   - name: coreDNSImageTag
     required: true
     schema:
       openAPIV3Schema:
         type: string
         default: ""
+        example: "1.8.5"
```

We have to have a `v` prefix. Let's also use v1.8.4 as that's the current default (full image `k8s.gcr.io/coredns/coredns:v1.8.4`)
```suggestion
        example: "v1.8.4"
```

---

Same in the other ClusterClass